easybuild/easyconfigs/f/fastqc/fastqc-0.11.2.eb

```diff
@@ -1,27 +0,0 @@
-easyblock = 'EB_Java'
-
-name = 'fastqc'
-version = '0.11.2'
-
-homepage = 'http://www.bioinformatics.babraham.ac.uk/projects/fastqc/'
-description = """A set of tools (in Java) for working with next generation sequencing data in the BAM format."""
-
-toolchain = {'name': 'dummy', 'version': ''}
-
-source_urls = ['http://www.bioinformatics.babraham.ac.uk/projects/fastqc/']
-sources = ['%(name)s_v%(version)s.zip']
-
-dependencies = [('Java', '1.7.0_60')]
-
-postinstallcmds = ["chmod -R a+rx %(installdir)s/fastqc %(installdir)s/run_fastqc.bat"]
-
-modextrapaths = {'PATH': ['']}
```

Comment:
@boegel - I thought the easyblock `Binary` which `PackedBinary` inherits, sets this by default?